### PR TITLE
Panel

### PR DESCRIPTION
**Pull request by KenMutuma**
@briankip review #457

### CODE REVIEW

**Comment by kiproping** [comment]
Could you check why the tests are failing.

**Comment by briankip** [comment]
Could you check why this test are failing

**Comment by KenMutuma** [comment]
@briankip review please

File: app/database/migrations/2014_07_24_082711_CreatekBLIStables.php [hunk 1]
```diff
@@ -117,17 +117,30 @@ public function up()
             $table->softDeletes();
             $table->foreign('measure_id')->references('id')->on('measures');
         });
-
+        
+         Schema::create('panel', function(Blueprint $table)
+        {
```
Comment (briankip) [inline app/database/migrations/2014_07_24_082711_CreatekBLIStables.php]: this need to be a new migration file. We are installing in already existing systems.

File: app/database/migrations/2014_07_24_082711_CreatekBLIStables.php [hunk 2]
```diff
@@ -117,17 +117,30 @@ public function up()
...
             $table->increments('id')->unsigned();
             $table->string('name', 100);
             $table->string('description', 100)->nullable();
             $table->integer('test_category_id')->unsigned();
+            $table->integer('panel_id')->unsigned();
             $table->string('targetTAT', 50)->nullable();
             $table->string('prevalence_threshold', 50)->nullable();
             
             $table->foreign('test_category_id')->references('id')->on('test_categories');
+            $table->foreign('panel_id')->references('id')->on('panel');
```
Comment (briankip) [inline app/database/migrations/2014_07_24_082711_CreatekBLIStables.php]: this too. All changes here need to be in a new file

**Comment by briankip** [comment]
Tests are also still failing.

**Comment by briankip** [comment]
You also need to add ability to add test to a panel

**Comment by briankip** [comment]
Also, delete button does not work.

**Pull request merged**